community/modules/scripts/pbspro-install/variables.tf

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "rpm_url" {
+  description = "Path to PBS Pro RPM file for select PBS host type (server, client, execution)"
+  type        = string
+}
+
+variable "pbs_exec" {
```

Comment:
nit: exec sounds like its pointing to the executable itself. Can this be updated to "pbs_exec_dir", "pbs_install_prefix" or similar?

Reply:
These values come directly from the PBS parameter names (a mix of environment variables, settings in `/etc/pbs.conf` and settable via qmgr). See section 3.5.2.2:

[bigbook]: https://2021.help.altair.com/2021.1/PBSProfessional/PBS2021.1.pdf